### PR TITLE
issue fixes mostly on form-views

### PR DESCRIPTION
**Pull request by nathan-tang**
// I think i can close these:
Issue #74
Issue #80
Issue #66
Issue #62
Issue #85
Issue #60
Issue #71
Issue #47
Issue #53
Issue #69
Issue #19

// iffy ones that will probably leave open for now
Issue #84 (sorta? I added a line break then deleted it but it fixed it???)
Issue #70 && Issue #48 (sizing is weird between different monitors probably will have to test this more)

### CODE REVIEW

File: app/client/src/app/components/form-components/educationalInfo.js [hunk 1]
```diff
@@ -99,7 +100,11 @@ export default function EducationalInfo(props) {
               as="select"
               name="major"
               className={
-                "form-control" + (errors && errors.major ? " error" : "")
+                "form-control" +
+                (errors && errors.major ? " error" : "") +
+                (values.pronouns !== undefined
```
Comment (zpinto) [inline app/client/src/app/components/form-components/educationalInfo.js:105]: shouldn't this be value.major

File: app/client/src/app/components/form-components/educationalInfo.js [hunk 2]
```diff
@@ -129,7 +134,10 @@ export default function EducationalInfo(props) {
               name="graduationYear"
               className={
                 "form-control" +
-                (errors && errors.graduationYear ? " error" : "")
+                (errors && errors.graduationYear ? " error" : "") +
+                (values.pronouns !== undefined
```
Comment (zpinto) [inline app/client/src/app/components/form-components/educationalInfo.js:138]: value.graduationYear

**Review by zpinto — CHANGES_REQUESTED** [review]
just change the ternary statements for making select items white and I think everything else is good

**Comment by zpinto** [comment]
The application form is not wide enough

Screen Shot 2020-12-09 at 12 05 27 PM

**Comment by nathan-tang** [comment]
latest commit addresses Issue #23 , no longer need to remove TODO as there should be no cases it is empty now.

**Review by zpinto — APPROVED** [review]
Nice job on this PR. You hit a bunch of issues 👍 I was able to figure out what was causing /application to not load and fixed it.

**Pull request merged**